packages/form-js-carbon-styles/src/carbon-styles.js

```diff
@@ -1156,14 +1162,6 @@ const CARBON_STYLES = css`
       padding: 0;
     }
 
-    .fjs-layout-column:first-of-type .fjs-form-field {
-      margin-left: 0;
-    }
-
-    .fjs-layout-column:last-of-type .fjs-form-field {
-      margin-right: 0;
-    }
-
```

Comment:
I think this is something that needs to be adjusted in tasklist directly, because what this does is basically stick the fields to the left and right boundaries of the form, which simply cannot work if groups have negative margins.

Reply:
So far the carbon-styles are mainly to set the correct styles to be embedded into Tasklist, I'm not sure whether we should do the alignments over there. The reason is that Carbon is pretty strict on the alignments so I'd expect we will have the same alignment requirements in other Carbonized products as well.

What about not removing the margins on `.fjs-form-field-group` elements?

Reply:
@pinussilvestrus @Skaiir Even though the original padding/margin comes from the Carbon grid, the external padding from the Carbon grid is an implementation detail from `form-js`
So right now I would have to always remove the padding because we have 3 different cases on Tasklist which 3 different paddings.

I would expect that when other teams, start using Carbon everywhere they might have different spacing. So I see no point in keeping the external margin if it will always be removed by everyone.

I think we should have no padding/margin on the outer containers. This way each user is free to set their own spacing in the `.fjs-container`

Reply:
@vsgoulart But the thing is, with no padding, we cannot display the groups at all. Those padding are *required* at least within the self-contained application to show everything. 

You can see in the below graphics that the outlines are cut off by the theoretical zero-margin. So they aren't as much of an implementation detail as they are a requirement. 

![grafik](https://github.com/bpmn-io/form-js/assets/17801113/497c61d9-3f9e-444b-868b-a6da394ac6b4)